tests/unit/dataactvalidator/utils.py

```diff
@@ -11,7 +11,7 @@ def insert_submission(db, submission):
     return submission.submission_id
 
 
-def error_rows(rule_file, staging_db, submission=None, models=None):
+def error_rows(rule_file, staging_db, submission=None, models=None, assert_num = None):
```

Comment:
I don't remember seeing the new `assert_num` parameter being used anywhere--do we still need this?